beps/0001-notifications-system/README.md

```diff
@@ -160,89 +168,202 @@ The following backend service interfaces are added as part of this proposal.
 ```ts
 // TODO - We may want to add an additional wrapping here with interfaces for Notification and NotificationParameters
 
-interface SendNotificationRequest {
-  receivers: SignalReceivers;
+export type NotificationRecipients =
```

Comment:
Missing `severity`

Reply:
Priority, severity, criticality? What's the way we go?

Reply:
Severity. We show impact of the notification not order of its next processing.

Reply:
In #22510 it was `priority` but let's go with `severity` then